<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RectificationGUI</class>
 <widget class="QMainWindow" name="RectificationGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1521</width>
    <height>1113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="coronal_gview">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="horizontal_gview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>841</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="sagittal_gview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>841</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sameX">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>860</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Same x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sameY">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>890</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Same y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sameZ">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>920</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Same z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_done">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>970</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hxy">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>570</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>580</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h_xy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>610</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h_yx</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hyx">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>600</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>640</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h_zx</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hzx">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>630</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>580</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h_xz</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hxz">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>570</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>610</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h_yz</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hyz">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>600</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>640</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>h_zy</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hzy">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>630</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1521</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
